src/Functions/like.h

```diff
@@ -11,7 +11,7 @@ struct NameLike
     static constexpr auto name = "like";
 };
 
-using LikeImpl = MatchImpl<NameLike, /*SQL LIKE */ true, /*revert*/false>;
+using LikeImpl = MatchImpl<NameLike, true, false, false>;
```

Comment:
I liked the comment here. Actually, right now it is hard to guess what those flags mean :)
Probably, we can use bit flags like `SQL_LIKE = 1; REVERT = 2; SOMETHING = 8` and use e.g. `MatchImpl<NameLike, SQL_LIKE | REVERT>`